src/native-common/RootView.tsx

```diff
@@ -125,17 +126,23 @@ abstract class BaseRootView<P extends BaseRootViewProps> extends React.Component
                 </RN.View>
                 { modalLayerView }
                 { popupLayerView }
-                <RN.View
-                    style={ _styles.liveRegionContainer }
-                    accessibilityLabel={ this.state.announcementText }
-                    accessibilityLiveRegion={ AccessibilityUtil.accessibilityLiveRegionToString(Types.AccessibilityLiveRegion.Polite) }
-                />
+                { announcerView }
             </RN.Animated.View>
         );
 
         return this.renderTopView(content);
     }
 
+    renderAnnouncerView(): JSX.Element {
```

Comment:
This should be a private or protected method and should begin with an underscore.